lib/graphql/language/token.rb

```diff
@@ -4,13 +4,15 @@ module Language
     # Contains type, value and position data.
     class Token
       # @return [Symbol] The kind of token this is
-      attr_reader :name
+      attr_reader :name, :prev_token, :line
+      attr_accessor :next_token
```

Comment:
If we do it this way in `get_description`: 

```ruby 
loop do 
  next_token = token 
  token = token.prev_token 
  # ...
  if next_token.line == token.line + 1 
    # ... 
  end 
end 
```

Could we drop `Token#next_token`?

If that's the only use for `next_token`, we might be able to save ourselves the assignment for nodes that don't need it!